accounts-journey/presentation/model/AccountSummaryUiModel.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  AccountSummaryUiModel.swift
+//  AccountsJourney
+//
+//  Created by George Nyakundi on 26/10/2023.
+//
+
+import Foundation
+
+struct AccountSummaryUiModel {
```

Comment:
Ui I shouldn't be uppercased? (I will not mention this everywhere, I noticed cause it's a abbreviation of Interface)

Reply:
😅 I'll replace this Ui with UI